zerver/openapi/zulip.yaml

```diff
@@ -10926,9 +10942,11 @@ paths:
                           - 1 = everyone
                           - 2 = members only
                           - 3 = administrators only
-                          - 4 = nobody (though note that administrators can change this setting).
+                          - 4 = nobody (note that administrators can change this setting)
```

Comment:
nit: I feel like "though" is doing useful work here. Without it, it seems a little hard to be sure how this line is meant to parse. (A sentence break between "nobody" and the "note that …" would also do it, but that's harder to fit in smoothly here.)

Reply:
That makes sense and I can add that back in. I think what I am really struggling with here is the lack of capitalization. It makes the whole section seem more informal (and therefore less clear?).

A nit that I have in the OpenAPI documentation in general is how inconsistent we are in descriptions when listing out enum int values to text. Sometimes we use "=>" and other times just "=". Sometimes the text is capitalized and other times not.

Reply:
I think we should look at whether OpenAPI has a recommended way to do enums; it seems likely to me that we could replace all these fixed lists with a more standard thing.

That said ... I think we're working towards replacing most of these enum type settings with user groups, and probably whatever fancy system OpenAPI has for enums will not work well for explaining legacy formats ... so it's perhaps better to just try to do something reasonable on this point. I don't have a strong preference on syntax here; could be a good chat.zulip.org discussion.